.gitignore

```diff
@@ -19,6 +19,44 @@ src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Actual/
 *.db
 *.opendb
 launchsettings.json
+[Dd]ebug/
+[Rr]elease/
+x64/
 
 # Workaround for an msbuild/dotnet bug on Linux
 src/BinSkimLinux.sln
+
+*_i.c
+*_p.c
+*_h.h
+*_i.h
+*.ilk
+*.meta
+*.obj
+*.pch
+*.pdb
```

Comment:
if we can use the folder ignore it would be better,

above:
`# Build results 
bld/
bin/
obj/`

this particular one *.pdb since we frequently checking in pdb files